projects/plugins/jetpack/extensions/index.json

```diff
@@ -50,7 +50,7 @@
 		"videopress/video",
 		"videopress/video-chapters"
 	],
-	"experimental": [ "anchor-fm", "premium-content", "conversation", "dialogue" ],
+	"experimental": [ "anchor-fm", "premium-content", "conversation", "dialogue", "writing-prompts" ],
```

Comment:
Note that including writing prompts as an experimental block will limit it to Simple and Atomic sites, and exclude it from self-hosted Jetpack (unless opted in using the `JETPACK_EXPERIMENTAL_BLOCKS` constant).

Reply:
> it seems strange to me that this is implemented as an experimental block

@roo2, see my comment above: this let's us create an editor extension that targets WP.com hosted sites. I think the editor extensions are intended to go in their own monorepo project, eventually, but still live within the main Jetpack plugin, for now.

Reply:
ok, makes sense to me 👍